index.html

```diff
@@ -211,5 +211,54 @@ <h1 class="about-heading">About me</h1>
         </li>
       </ul>
     </section>
+    <section class="contact-section">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form class="form-control">
+        <input
+          placeholder="Full Name"
+          class="text-field"
+          type="text"
+          minlength="30"
+          required
+        />
+        <input
+          placeholder="Email Address"
+          class="text-field"
+          type="email"
+          required
+        />
+        <textarea
+          class="message-field"
+          placeholder="Enter text here"
+          minlength="500"
```

Comment:
- PLease from the requirements the length of a Textarea field  should have a max length of 500, please insert the correct fix